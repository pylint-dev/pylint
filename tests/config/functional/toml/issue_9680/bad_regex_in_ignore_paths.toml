# Check that we report regex error in configuration file properly
[tool.pylint."main"]
ignore-paths = ['project/tooling_context/**']
