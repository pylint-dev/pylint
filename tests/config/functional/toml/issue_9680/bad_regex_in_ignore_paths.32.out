usage: pylint [options]
pylint: error: argument --ignore-paths: Error in provided regular expression: project\\tooling_context\\**|project/tooling_context/** beginning at index 27: multiple repeat
